src/javascripts/ng-admin/Crud/list/maDatagrid.js

```diff
@@ -12,14 +12,10 @@ define(function (require) {
             template: datagridView,
             scope: {
                 name: '@',
-                entries: '=',
+                entries: '&',
```

Comment:
`entries` can't be in a one-way binding.
When doing infinite pagination, we have to update `entities` array to display next entities in the list.